__tests__/src/actions/window.test.js

```diff
@@ -12,14 +12,18 @@ describe('window actions', () => {
 
       const mockState = {
         companionWindows: {},
-        windows: {
+        elasticLayout: {
           window: {
```

Comment:
Should there be another `windows` layer here?

Reply:
I don't think so; `window` is just a terrible name for the window id there.